content/docs/guides/crosswalk/kubernetes/aws/01-cloud-stacks/01-identity.md

```diff
@@ -0,0 +1,118 @@
+---
+
+title: Identity
+
+aliases: ["/docs/guides/crosswalk/kubernetes/aws/identity/"]
```

Comment:
Can you explain this alias? Did we used to have content at this URL path?

Reply:
See https://github.com/pulumi/docs/pull/1850#discussion_r335169471